marketplace/plugins/harperdb/lib/index.ts

```diff
@@ -0,0 +1,75 @@
+import { QueryError, QueryResult, QueryService, ConnectionTestResult } from '@tooljet-marketplace/common';
+import { SourceOptions, QueryOptions } from './types';
+import harperive from 'harperive';
+import JSON5 from 'json5';
+
+export default class Harperdb implements QueryService {
+  async run(sourceOptions: SourceOptions, queryOptions: QueryOptions, dataSourceId: string): Promise<QueryResult> {
+    const harperdbClient = await this.getConnection(sourceOptions);
+    const { operation, mode } = queryOptions;
+    let result = {};
+
+    try {
+      if (mode === 'sql') {
+        const { sql_query } = queryOptions;
+        const sql_operation_response = await harperdbClient.query(sql_query);
```

Comment:
please camel cases

Reply:
I have updated the Logic bit. Like instead of storing response in a `variable` - `sql_operation_response `, now directly storing it in result. Only on final return data has been taken and sent like `data: result?.data ?? {}`